seatunnel-e2e/seatunnel-connector-v2-e2e/connector-rocketmq-e2e/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove logback config file

seatunnel using log4j2

Reply:
> remove logback config file
> 
> seatunnel using log4j2

To disable the debug log
https://www.testcontainers.org/supported_docker_environment/logging_config/

Reply:
> > remove logback config file
> > seatunnel using log4j2
> 
> To disable the debug log https://www.testcontainers.org/supported_docker_environment/logging_config/

@hailin0  At present, all tests have been completed. Logback-test is mainly to disable some debug logs. Do you want to continue to modify here? Or there is a better way to deal with it

Reply:
logback is adapted to log4j2 through slf4j bridge, you can configure log4j2 logger
@sunxiaojian

Reply:
The easiest way is to delete the logback configuration

Reply:
> logback is adapted to log4j2 through slf4j bridge, you can configure `log4j2-test.properties` logger @sunxiaojian

fixed